libs/entry-components/styles/modules/vendors/angular-material/_typography-generator.scss

```diff
@@ -23,6 +23,12 @@
 	$typography: -set-font-size-for('titles', $typography, $fonts, ('subtitle-1', 'subtitle-2'));
 	$typography: -set-font-family($typography, $fonts);
 
```

Comment:
-set-font-size-for and -set-letter-spacing-for are identical methods, so they should be joined in the following way:
1) get $letter-spacing from theme like we do for fonts
2) create third method called -set-font-for with the same parameters as for set-font-size-for and inject $letter-spacing to fourth argument
3) In that new method c/p code from -set-font-size-for and on the first line get letter-spacing as you did in -set-letter-spacing-for
4) First do @each and inside @each add ifs and optionally add font-size and letter-spacing if desired values aren't null